docs/obs/traffic-inspection.mdx

```diff
@@ -28,12 +28,32 @@ ngrok offers multiple ways to inspect the traffic going to your service.
 :::note
 
 ngrok Traffic Inspector is currently in developer preview and only available to
-specific customers. While in developer preview, you will not be charged anything additional and things will change and break
+specific customers. While in developer preview, you will not be charged for its usage, things will change and break
 without notice, and data may be lost at any time. If you have feedback or questions
 please [reach out to the product team](mailto:support@ngrok.com?subject=Traffic%20Insights%20Feedback).
 
 :::
 
 ## Full Capture mode
 
-Coming soon. If you need to view request and response bodies, in the meantime, you can leverage the [ngrok Agent Local Traffic Inspector](/docs/agent/web-inspection-interface/#inspecting-requests) which is available in the standalone agent.
+By default, the ngrok Traffic Inspector will only store the metadata about each request and response. Full Capture mode is an opt-in setting on your ngrok account that will store the full request and response parameters, headers, and bodies for each traffic event that flows through your endpoints.
+This can be enabled by account admins by visiting the Observability section in the [ngrok Settings Page](https://dashboard.ngrok.com/settings) after logging in.
```

Comment:
We could link them straight there:

```suggestion
This can be enabled by account admins by visiting the Observability section in the [ngrok Settings Page](https://dashboard.ngrok.com/settings#observability) after logging in.
```